subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/verification/DependencyVerificationSignatureCheckIntegTest.groovy

```diff
@@ -62,34 +62,6 @@ class DependencyVerificationSignatureCheckIntegTest extends AbstractSignatureVer
         succeeds ":compileJava"
     }
 
-    def "doesn't need checksums if signature is verified and trust using long id"() {
```

Comment:
Test is not required anymore, as we cannot use long IDs before - verified in other tests.